src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderContact/CheckProviderContactViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.Roatp.CourseManagement.Web.Models.ProviderContact;
+
+public class CheckProviderContactViewModel : IBackLink
+{
+    public string BackUrl { get; set; }
```

Comment:
If the backurl is defaulted to browser back, why are we enforced to implement this in each model. Can we use a different interface for browser back ?

Reply:
I think I get what you mean, but wouldn't mind a few minutes to discuss

Reply:
9 pages in all, and some of them are in the ProviderLocations journeys....